src/table/__tests__/empty-state.test.tsx

```diff
@@ -61,19 +62,42 @@ function findMergedCell(wrapper: TableWrapper) {
   return wrapper.findByClassName(styles['cell-merged']);
 }
 
+function mockResizeObserver(contentBoxWidth: number) {
+  const callbacks: ((entry: ContainerQueryEntry) => void)[] = [];
+  const fireCallbacks = (entry: ContainerQueryEntry) => callbacks.forEach(cb => cb(entry));
+
+  jest.mocked(useResizeObserver).mockImplementation((_target, cb) => {
+    // The table uses more than one resize observer.
+    // The callback must be triggered for all to ensure the expected one is targeted as well.
+    callbacks.push(cb);
+
+    useEffect(() => {
+      cb({ contentBoxWidth } as unknown as ContainerQueryEntry);
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    });
```

Comment:
In the other PR you had and empty array, here you do not. Which one is correct?

Reply:
Thanks, I removed it accidentally. Having an empty dependency array is correct